bread/contrib/reports/urls.py

```diff
@@ -45,14 +47,76 @@ def get_layout(self):
         return ret
 
 
-def exceldownload(request, report_pk: int):
-    report = get_object_or_404(Report, pk=report_pk)
+class ReadView(views.ReadView):
+    def get_layout(self):
+
+        # ordering, copied from bread.views.browse.BrowseView.get_queryset
+        qs = self.object.filter.queryset
+        order = self.request.GET.get("ordering")
+        if order:
+            if order.endswith("__int"):
+                order = order[:-5]
```

Comment:
where does the literal `5` come from?

Reply:
:D good question
Stripping the ```"__int"``` from the argument. I could write order[:len("__int")]
In any case: It would be nice to improve this feature in the future (by having a more generic approach) to allow for other sorting criterias as well. But caseless string sort and decimal sort solve 90% of the base cases I think.

Reply:
or ```order.rpartition("__int")[0]```

Reply:
or ```''.join(order.rsplit('__int', 1))```

Reply:
Python has just to many string functions ;)

Reply:
😄 I like `order[:-len("__int")]`! `rsplit` and `rpartition` are a bit less mainstream than slicing, so I would prefer the slicing.

Reply:
Agreed :)